src/test/java/com/Bai18.java

```diff
@@ -0,0 +1,54 @@
+package com;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.safari.SafariDriver.WindowType;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.utils.BasicTest;
+
+public class Bai18 extends BasicTest {
+
+    @Test()
+    public void bai18() throws Exception {
+        // Log in
+        String url = "https://bantheme.xyz/hathanhauto/tai-khoan/";
+        driver.get(url);
+        WebElement email = driver.findElement(By.xpath("//input[@id='username']"));
+        email.sendKeys("nhatthuyip@gmail.com");
+        WebElement password = driver.findElement(By.xpath("//input[@id='password']"));
+        password.sendKeys("Bgr2ftvvYsY8hVe");
+        WebElement btnLogin = driver.findElement(By.xpath("//button[contains(text(),'Đăng nhập')]"));
+        btnLogin.click();
+
+        // Get the current tab
+        String firstTabHandle = driver.getWindowHandle();
+
+        // Open a new tab using JavaScript
+        ((JavascriptExecutor) driver).executeScript("window.open();");
+
+        // Switch new tab
+        for (String handle : driver.getWindowHandles()) {
```

Comment:
khuc nay em dau can dung for loop dau, muon switch qua new tab e chi can get cai handle cuoi' cung trong cai lisy getWindowHandles no se nhanh hon

Reply:
e moi sua roi a. a xem giup em